code/datums/supply_packs/_supply_packs.dm

```diff
@@ -12,13 +12,17 @@ var/list/all_supply_groups = list(
 	"Attachments",
 	"Ammo",
 	"Weapons Specialist Ammo",
-	"Armor",
+	"Restricted equipement",
 	"Clothing",
 	"Medical",
 	"Engineering",
-	"Science",
+	"Research",
 	"Supplies",
 	"Food",
+	"Gear",
+	"Mortar",
+	"Explosive",
+	"Reagent tank",
```

Comment:
I think `Explosives` and `Reagent tanks` would be a better category since you have / will have more than one thing in them. Similar to how we have `Attachments`.

Reply:
the requested change have been made